assets/queries/terraform/aws/alb_not_dropping_invalid_headers/query.rego

```diff
@@ -0,0 +1,35 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource[name]
+	types := {"aws_alb", "aws_lb"}
+	name == types[x]
+	res := resource[m]
+	res.load_balancer_type == "application"
+	res.drop_invalid_header_fields == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("%s[{{%s}}].drop_invalid_header_fields", [types[x], m]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("%s[{{%s}}].drop_invalid_header_fields is set to true", [types[x], m]),
+		"keyActualValue": sprintf("%s[{{%s}}].drop_invalid_header_fields is set to false", [types[x], m]),
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i].resource[name]
+	types := {"aws_alb", "aws_lb"}
+	name == types[x]
+	res := resource[m]
+	res.load_balancer_type == "application"
```

Comment:
We need to execute this rule when `object.get(res, "load_balancer_type", "undefined") == "undefined"`, since `application` is the default value, as you can see [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb#load_balancer_type)

Reply:
good job